.travis.yml

```diff
@@ -11,27 +11,30 @@ cache: pip
 
 env:
   global:
+    - BASEDIR="https://raw.githubusercontent.com/open-contracting/standard-maintenance-scripts/master"
     - DISPLAY=:99.0
     # https://github.com/SeleniumHQ/docker-selenium/issues/87 :
     - export DBUS_SESSION_BUS_ADDRESS=/dev/null
 
 install:
+  - curl -s -S --retry 3 $BASEDIR/tests/install.sh | bash -
   - "pip install -r requirements_dev.txt"
   - "sudo apt-get -y purge chromium-browser"
   - "wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb"
   - "sudo dpkg -i google-chrome*.deb"
   - "sleep 3"
   - "google-chrome &"
-  - "wget https://chromedriver.storage.googleapis.com/75.0.3770.90/chromedriver_linux64.zip; unzip chromedriver_linux64.zip -d chromedriver; export PATH=$PATH:`pwd`/chromedriver"
+  - "export LATEST_CHROMEDRIVER=$(wget -q -O - http://chromedriver.storage.googleapis.com/LATEST_RELEASE)"
+  - "wget http://chromedriver.storage.googleapis.com/$LATEST_CHROMEDRIVER/chromedriver_linux64.zip; unzip chromedriver_linux64.zip -d chromedriver; export PATH=$PATH:`pwd`/chromedriver"
 
 
 before_script:
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16"
   - sleep 3 # give xvfb some time to start
-  - set -e # make sure we exit if one command fails
 
 script:
+  - curl -s -S --retry 3 $BASEDIR/tests/script.sh | bash -
   - "google-chrome --version"
   - "google-chrome &"
-  - "DJANGO_SETTINGS_MODULE=cove_project.settings  python3 -m pytest ./tests/"
-  - "flake8"
```

Comment:
@jpmckinney what's the rationale for removing flake8 from the automated test? On our stuff, we've found it a helpful way of catching anything that slips through - eg forgetting to run flake8 after fixing other test fails.

Reply:
It's contained in `curl -s -S --retry 3 $BASEDIR/tests/script.sh | bash -` which is run on all other OCP repositories.